src/modules/Elsa.Workflows.Sink/Elsa.Workflows.Sink.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
```

Comment:
To include common settings for Elsa library projects distributed as NuGet packages, please include these sections at the top of the file (but inside the <Project> element):

```xml
<Import Project="..\..\..\common.props" />
<Import Project="..\..\..\configureawait.props" />
```

the common props already enable Nullable, so you can remove that one.
We also need to include a description and tags.

Here's an example:

```xml
<Project Sdk="Microsoft.NET.Sdk">

    <Import Project="..\..\..\common.props" />
    <Import Project="..\..\..\configureawait.props" />

    <PropertyGroup>
        <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
        <Description>
            Provides workflow state sinks that store workflow state in various formats and locations.
        </Description>
        <PackageTags>elsa module</PackageTags>
    </PropertyGroup>
...
```